client/src/pages/Home/FooterNav/FooterNav.css

```diff
@@ -0,0 +1,72 @@
+#footernav{
+    font-family: "Ambit SemiBold";
+    font-size: 16px;
+    color: rgb(199, 202, 202);
+    /* color: var(--main-bg-color); */
+    background-color: var(--darkgreen-color);
+}
+
+.title{
+    font-size: 19px;
+    color: rgb(206, 242, 248);
+}
+
+.footernav_container{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 100px 150px 50px 150px;
+    
+}
+
+.nav_container{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 45%;
+}
+
+.nav-box{
+    display: flex;
+    flex-direction: column;
+}
+
+a{
+    text-decoration: none;
+    padding-top: 20px;
+    color: rgb(199, 202, 202);
+}
+
+a:hover {
```

Comment:
you should not target the anchor element directly instead you should use class name to update this element other wise your style will effect all <a> elements.